src/AutoMapper/Configuration/MappingExpression.cs

```diff
@@ -540,8 +541,11 @@ public void Configure(TypeMap typeMap)
                 var attrs = destProperty.GetCustomAttributes(true);
                 if (attrs.Any(x => x is IgnoreMapAttribute))
                 {
-                    IgnoreDestinationMember(destProperty);
-                    _reverseMap?.ForMember(destProperty.Name, opt => opt.Ignore());
+                    IgnoreDestinationMember(destProperty);
+                    if (typeMap.SourceType.GetInheritedMember(destProperty.Name) != null)
+                    {
+                        _reverseMap?.ForMember(destProperty.Name, opt => opt.Ignore());

```

Comment:
No, I don't. here the destination of (reverse) mapping is the source. I should ignore source property with the same **name** as `destProperty.Name` **if exists**.
`IgnoreDestinationMember` takes a `MemberInfo` as a parameter, not a **probably existing** property name. I can not get a `MemberInfo` from source if it does not exist there. I have to check for its existence first.
line 544 ignores in the forward direction. line 545 checks for source property existence, line 547 ignores source property. if this is wrong, what is the right way?
what do you mean by saying 

> You already have the member